todo_app/app.py

```diff
@@ -1,15 +1,22 @@
-from flask import Flask
-
-from todo_app.flask_config import Config
```

Comment:
If you did want to stop using flask_config, then you should delete flask_config.py otherwise you are keeping unused code around.